.github/scripts/trymerge.py

```diff
@@ -579,46 +578,58 @@ def read_merge_rules(repo: GitRepo) -> List[MergeRule]:
     return cast(List[MergeRule], rc)
 
 
-
 def find_matching_merge_rule(pr: GitHubPR, repo: GitRepo, force: bool = False) -> MergeRule:
     """Returns merge rule matching to this pr or raises an exception"""
     changed_files = pr.get_changed_files()
     approved_by = set(pr.get_approved_by())
     rules = read_merge_rules(repo)
+    reject_reason = f"PR {pr.pr_num} does not match merge rules"
+    #  Used to determine best rejection reason
+    # Score 0 to 10K - how many files rule matched
+    # Score 10K - matched all files, but no overlapping approvers
+    # Score 20K - matched all files and approvers, but lacks mandatory checks
+    reject_reason_score = 0
```

Comment:
```suggestion
    num_matching_files = 0
```
reject_reason_score is a confusing name here

Reply:
oh nvm i see